desktop/cmp/panel/impl/PanelHeader.ts

```diff
@@ -28,7 +29,7 @@ export const panelHeader = hoistCmp.factory({
         const onDoubleClick = () => {
             if (isModal) {
                 panelModel.toggleIsModal();
-            } else {
+            } else if (panelModel.collapsible) {
```

Comment:
Good catch!  I see we currently throw an exception to the console if you double-click the header of a non-collapsible panel.